arrow/arrow.py

```diff
@@ -1012,42 +1012,42 @@ def humanize(
                 if diff < 10:
                     return locale.describe("now", only_distance=only_distance)
 
-                if diff < 45:
+                if diff < self._SECS_PER_MINUTE:
                     seconds = sign * delta
                     return locale.describe(
                         "seconds", seconds, only_distance=only_distance
                     )
 
-                elif diff < 90:
+                elif diff < self._SECS_PER_MINUTE * 2:
                     return locale.describe("minute", sign, only_distance=only_distance)
-                elif diff < 2700:
-                    minutes = sign * int(max(delta / 60, 2))
+                elif diff < self._SECS_PER_HOUR:
+                    minutes = sign * int(max(delta / self._SECS_PER_MINUTE, 2))
                     return locale.describe(
                         "minutes", minutes, only_distance=only_distance
                     )
 
-                elif diff < 5400:
+                elif diff < self._SECS_PER_HOUR * 2:
                     return locale.describe("hour", sign, only_distance=only_distance)
-                elif diff < 79200:
-                    hours = sign * int(max(delta / 3600, 2))
+                elif diff < self._SECS_PER_DAY:
+                    hours = sign * int(max(delta / self._SECS_PER_HOUR, 2))
                     return locale.describe("hours", hours, only_distance=only_distance)
 
                 # anything less than 48 hours should be 1 day
```

Comment:
Should we put more of these comments in each case? E.g. anything less than 2 weeks is a week? Or do you think the code is clear enough and we can just remove this comment?

Reply:
I think we can remove it.